database_table.sql

```diff
@@ -0,0 +1,112 @@
+CREATE TABLE organizations (
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(255) NOT NULL,
+  created_at TIMESTAMP NOT NULL DEFAULT NOW(),
+  updated_at TIMESTAMP NOT NULL DEFAULT NOW()
+);
+
+-- Members table
+CREATE TABLE members (
+  id SERIAL PRIMARY KEY,
+  username VARCHAR(255) NOT NULL,
+  email VARCHAR(255) NOT NULL,
+  password VARCHAR(255) NOT NULL,
+  created_at TIMESTAMP NOT NULL DEFAULT NOW(),
+  updated_at TIMESTAMP NOT NULL DEFAULT NOW()
+);
+
+-- Organization members table
+CREATE TYPE roles as ENUM('organization_head', 'member');
+CREATE TABLE organization_members (
+  organization_id INT REFERENCES organizations(id),
+  member_id INT REFERENCES members(id),
+  organization_member_role roles ,
+  created_at TIMESTAMP NOT NULL DEFAULT NOW(),
+  updated_at TIMESTAMP NOT NULL DEFAULT NOW()
+);
+-- Plans table
+CREATE TABLE plans (
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(255) NOT NULL,
+  duration INT NOT NULL,
+  created_at TIMESTAMP NOT NULL DEFAULT NOW(),
+  updated_at TIMESTAMP NOT NULL DEFAULT NOW()
+);
+
+-- Organization plans table
+CREATE TABLE organization_plans (
```

Comment:
Can organisation have more than 1 plan?

Reply:
Have added a UNQIUE keyword to organization_id